src/main/java/ch/njol/skript/expressions/ExprYawPitch.java

```diff
@@ -71,26 +84,60 @@ public Number convert(final Object object) {
 	@SuppressWarnings({"null"})
 	@Override
 	public Class<?>[] acceptChange(final ChangeMode mode) {
-		if (mode == ChangeMode.SET || mode == ChangeMode.ADD || mode == ChangeMode.REMOVE)
+		if (mode == ChangeMode.SET || mode == ChangeMode.ADD || mode == ChangeMode.REMOVE) {
+			if (Player.class.isAssignableFrom(getExpr().getReturnType()) && !PLAYER_SUPPORT) {
+				Skript.error("Changing a rotation of a player requires Paper 1.19 and above.");
+				return null;
+			}
 			return CollectionUtils.array(Number.class);
+		}
 		return null;
 	}
 
 	@SuppressWarnings("null")
 	@Override
-	public void change(Event e, Object[] delta, ChangeMode mode) {
+	public void change(Event event, Object[] delta, ChangeMode mode) {
 		if (delta == null)
 			return;
 		float value = ((Number) delta[0]).floatValue();
-		for (Object single : getExpr().getArray(e)) {
-			if (single instanceof Location) {
+		for (Object single : getExpr().getArray(event)) {
```

Comment:
`obj` is clearer here
```suggestion
		for (Object obj : getExpr().getArray(event)) {
```